gql_build/example/README.md

```diff
@@ -1,37 +1,40 @@
-Add `gql_code_gen` to your `pubspec.yaml`
+Add `gql_build` to your `pubspec.yaml`
```

Comment:
Would you mind creating a quick little PR containing my mistakes in this file?

I'm tempted to fix it myself, but I'd love to give you credit in git history.

Reply:
done https://github.com/gql-dart/gql/pull/34